drivers/spi/Kconfig.silabs_eusart

```diff
@@ -12,3 +12,13 @@ config SPI_SILABS_EUSART
 	select PINCTRL if SOC_FAMILY_SILABS_S2
 	help
 	  Enable the EUSART SPI driver
+
+config SPI_SILABS_EUSART_DMA
+	bool "Silabs EUSART SPI controller driver DMA enabled transfer"
+	depends on SPI_SILABS_EUSART
+	select DMA
+	default y if SPI_ASYNC
+
+config SPI_DMA_MAX_TRANSFER_BLOCK_PER_CHANNEL
```

Comment:
Should this option have `SILABS_EUSART` in it? Could be something like `SPI_SILABS_EUSART_DMA_MAX_BLOCKS`.
It should also depend on the above config such that it's only available if DMA is enabled.